index.js

```diff
@@ -246,3 +289,29 @@ bot.onText(/^\//, async msg => {
     }
   );
 });
+
+//Check if new user introduced himself or not
+//Cron Job to carry the check every 2 hour
+cron.schedule("0 */2 * * *", async () => {
+  await NewUser.find({ introduction: false }, (err, users) => {
+    if (err) {
+      console.log(err);
+    }
+    users.map(usr => {
+      console.log(usr);
+      let d = new Date();
+      const DAY = 86400000;
+      if (d.getTime() - usr.JoinDate >= DAY) {
+        console.log("Kicking user ", usr.UserName);
+        bot.kickChatMember(usr.ChatID, usr.UserId);
+      } else if (d.getTime() - usr.JoinDate >= DAY / 2) {
+        bot.sendMessage(
+          usr.ChatID,
+          "WARNING for @" +
+            usr.UserName +
+            "\nIntroduce yourself within the next 12 hours or you will be kicked"
```

Comment:
Add `Kindly` before Introduce yourself... The manner of these statements matter a lot.